File: fizzbuzz.php
#!/usr/bin/php
<?php
    // Language:       php
    // Web site:       https://www.php.net/
    // Last tested on: Ubuntu 22.04.4
    // Requires:       apt-get install php5 php5-cli
    for ($i = 1; $i <= 100; $i ++) {
        if ($i % 15 == 0) {
            echo "FizzBuzz\n";
        }
        elseif ($i % 3 == 0) {
            echo "Fizz\n";
        }
        elseif ($i % 5 == 0) {
            echo "Buzz\n";
        }
        else {
            echo "$i\n";
        }
    }
?>

File: fizzbuzz.lua
#!/usr/bin/lua

-- Language:       Lua
-- Web site:       http://www.lua.org/
-- Last tested on: Ubuntu 22.04.5
-- Requires:       apt-get install lua5.3
--                 update-alternatives --install /usr/bin/lua  lua-interpreter /usr/bin/lua5.3  90
--                 update-alternatives --install /usr/bin/luac lua-compiler    /usr/bin/luac5.3 90

for i = 1, 100 do
    if i % 15 == 0 then
        print("FizzBuzz")
    elseif i % 3 == 0 then
        print("Fizz")
    elseif i % 5 == 0 then
        print("Buzz")
    else
        print(i)
    end
end
